File: app/src/main/java/com/example/myapplication/MyPreferences.java
package com.example.myapplication;

import android.os.Bundle;
import android.preference.PreferenceActivity;

public class MyPreferences extends PreferenceActivity {
    @Override
    public void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        getPreferenceManager().setSharedPreferencesName("my_prefs");
        addPreferencesFromResource(R.xml.my_preferences);
    }
}

File: app/src/main/java/com/frogobox/pianotilesclone/MyPreferences.kt
package com.frogobox.pianotilesclone

import android.preference.PreferenceActivity
import android.os.Bundle
import com.example.myapplication.R

class MyPreferences : PreferenceActivity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        preferenceManager.sharedPreferencesName = "my_prefs"
        addPreferencesFromResource(R.xml.my_preferences)
    }

}